src/python/bot/tasks/train_gradientfuzz_task.py

```diff
@@ -13,9 +13,297 @@
 # limitations under the License.
 """GradientFuzz training task."""
 
+from builtins import str
+
+import glob
+import os
+import shutil
+import sys
+
+from bot.fuzzers.ml.gradientfuzz import constants
+from bot.fuzzers.ml.gradientfuzz import run_constants
+from build_management import build_manager
+from fuzzing import corpus_manager
+from google_cloud_utils import storage
 from metrics import logs
+from system import archive
+from system import environment
+from system import new_process
+from system import shell
+
+# Model script directory.
+GRADIENTFUZZ_SCRIPTS_DIR = os.path.join(
+    os.path.dirname(os.path.dirname(__file__)), 'fuzzers', 'ml', 'gradientfuzz')
+
+
+def get_corpus(corpus_directory, fuzzer_name):
+  """Get corpus directory.
+
+  This function will download latest corpus backup file from GCS, unzip
+  the file and put them in corpus directory.
+
+  TODO(ryancao): Make default corpus directory.
+
+  Args:
+    directory: The directory to place corpus.
+    fuzzer_name: Fuzzer name, e.g. libpng_read_fuzzer, xml_parser_fuzzer, etc.
+
+  Returns:
+    True if corpus can be acquired, False otherwise.
+  """
+  # e.g. clusterfuzz-libfuzzer-backup
```

Comment:
Nit: probably remove these comments. I think these are reasonable enough for folks familiar with the CF codebase, and can be looked up pretty easily if not. I also wouldn't be opposed to a more detailed comment, but don't think that's necessary. This just feels like it doesn't quite do enough on its own.

Reply:
Gotchu. Removed!